app/jobs/daily_meetup_crawl_job.rb

```diff
@@ -7,19 +7,18 @@ def perform(*args)
     @meet = meetup_api.method_request('self/calendar', {})
 
     @meet.each do |list|
-      if list["venue"]
-      url = list["link"]
-      address = list["venue"]["name"] + ", " + list["venue"]["address_1"] + ", " + list["venue"]["city"]
-      title = list["name"]
-      time = Time.at(list["time"]/1000).strftime("%Y-%m-%d %I:%M:%S")
-      event = Event.new
-      event.meetup_title = title
-      event.start_time = time
-      event.meetup_url = url
-      event.location = address
-      event.save
+        if list["venue"]
```

Comment:
two things:
1- is this indented too much?
2- Why do we check if there is a venue for the meetup?